src/com/flower/kiev/FlowerStore.java

```diff
@@ -0,0 +1,79 @@
+package com.flower.kiev;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FlowerStore {
+
+    private int wallet;
+    private Flower[] flowers;
+
+    public int getWallet() {
+        return wallet;
+    }
+
+    public void printFlowers() {
+        for (int i = 0; i < flowers.length; i++) {
+            System.out.print(flowers[i].getClass().getSimpleName() + " ");
```

Comment:
As in this class, you have just default constructor first value of `flowers` would be null. And if you invoke printFlowers() you will get NPE.
Also, I believe, you really don't need field   `flowers`. Just use the local variable.